Makefile

```diff
@@ -118,8 +118,14 @@ itest-only:
 	@$(call print, "Running integration tests with btcd backend.")
 	$(ITEST_BTCD)
 
+itest-only-neutrino:
```

Comment:
Seems like it'd be better to use a `backend` option in the `Makefile`? Otherwise we start having all these duplicate targets just for different backends.

Reply:
Yes, good idea! Done, and it's much cleaner :)